db/db_basic_test.cc

```diff
@@ -1287,6 +1287,48 @@ TEST_F(DBBasicTest, MultiGetBatchedMultiLevel) {
   }
 }
 
+TEST_F(DBBasicTest, MultiGetPrefixExtractor) {
```

Comment:
P.S. As it stands right now, we're only testing FullFilterBlockReader here; we could extend this so it also tests PartitionedFilterBlockReader (which I believe uses the base class implementation of PrefixesMayMatch).